packages/tools/e2e-tests/src/page-objects/react-ui/card.component.ts

```diff
@@ -0,0 +1,24 @@
+import type { Page } from '@playwright/test';
+
+export default class CardComponent {
+  private readonly _page: Page;
+
+  public constructor(page: Page) {
+    this._page = page;
+  }
+
+  public async setValueForTextbox(label: string, value: string): Promise<void> {
```

Comment:
lol, reminds me of https://github.com/kadena-community/kadena.js/blob/90a96dd0e9a0ec6db7c5784a261ccc0e99bd91e4/packages/tools/lint-package/.eslintrc.cjs#L9-L10

Reply:
There are quite a few functions that simply don't _need_ to return anything. I personally like to make this explicit though.